packages/page-accounts/src/Sidebar/JudgementTag.tsx

```diff
@@ -0,0 +1,44 @@
+// Copyright 2017-2021 @polkadot/app-accounts authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import type { Judgement } from '@polkadot/react-hooks/types';
+
+import React, { useMemo } from 'react';
+
+import { AddressSmall, Menu, Popup, Tag } from '@polkadot/react-components';
+
+import { getJudgementColor } from '../util';
+
+interface Props {
+  judgement: Judgement
+}
+
+function JudgementTag ({ judgement: { judgementName, registrars } }: Props): React.ReactElement<Props> {
+  const judgementColor = useMemo(() => getJudgementColor(judgementName), [judgementName]);
+
+  return (
+    <Popup
```

Comment:
This is very weird - I can see the code, I can see your screenshot, but I don't get this info on my screen at all. Hover over the "Reasonable" and it is just, well, a cursor, no popup :)

Reply:
Some rebuild perhaps? With `yarn clean` and `yarn build`?

Reply:
The popup shows on click not on hover, did you try to click it? 😀

Reply:
Ahhh... yes, not hover, click. (Not quite obvious to me :))